### PR TITLE
fix uniqueserviceselector behavior if service selector is missing

### PR DESCRIPTION
**Pull request by bovy89**
**How to reproduce**
- default namespace with kubernetes base service (no selector)
- add network policy (no selector)
---> violation

**Expected result**
No constraint violation. Only services must be considered

### CODE REVIEW

**Review by maxsmythe — APPROVED** [review]
LGTM

**Comment by maxsmythe** [comment]
@bovy89 could you fix the DCO check?

**Comment by bovy89** [comment]
> @bovy89 could you fix the DCO check?

done

**Comment by maxsmythe** [comment]
@shomron @ritazh @sozercan LGTY?

File: src/general/uniqueserviceselector/src.rego [hunk 1]
```diff
@@ -29,7 +29,7 @@ violation[{"msg": msg}] {
   input.review.kind.version == "v1"
   input.review.kind.group == ""
   input_selector := flatten_selector(input.review.object)
```
Comment (ritazh) [inline src/general/uniqueserviceselector/src.rego:31]: We should check if input_selector is empty here.

**Review by ritazh — APPROVED** [review]

**Pull request merged**